### PR TITLE
(feat) Update the patient info header component to the new design 

### PR DESCRIPTION
**Pull request by Jexsie**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

- Update the patient info header to the new desgin 

### Before
image

### After
image

### CODE REVIEW

**Comment by denniskigen** [comment]
Is this work still ongoing, @Jexsie? If so, could you upload some before and after screenshots?

File: packages/esm-patient-allergies-app/src/allergies/allergies-tile.component.tsx [hunk 1]
```diff
@@ -10,27 +10,17 @@ interface AllergyTileInterface {
...
   if (allergies?.length) {
     return (
-      <div>
-        <p className={styles.label}>{t('allergies', 'Allergies')}</p>
-        <p className={styles.content}>
-          <span className={styles.value}>{allergies?.map((allergy) => allergy?.display).join(', ')}</span>
-        </p>
+      <div className={styles.content}>
+        {allergies?.map((allergy) => (
+          <Tile className={styles.tile}>{allergy?.display}</Tile>
```
Comment (denniskigen) [inline packages/esm-patient-allergies-app/src/allergies/allergies-tile.component.tsx:19]: Shouldn't these be [tags](https://react.carbondesignsystem.com/?path=/docs/components-tag--overview) instead?

File: packages/esm-patient-allergies-app/src/allergies/allergies-tile.scss [hunk 2]
```diff
@@ -10,8 +10,25 @@
...
+  color: #943d00;
+  text-align: center;
+  min-height: unset;
+  min-width: unset;
+  height: 1.5rem;
+  padding: 0 0.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 12px;
```
Comment (denniskigen) [inline packages/esm-patient-allergies-app/src/allergies/allergies-tile.scss:33]: You should get all of this styling for free when using Carbon tags.

**Comment by njiddasalifu** [comment]
hey @Jexsie is there still ongoing work here?

**Comment by Jexsie** [comment]
not really @njiddasalifu. Just some CI fails, am looking to into. Thanks

**Comment by denniskigen** [comment]
Closing as stale.

**Pull request closed without merging**